run `make load-secrets` through a terminal session on your laptop or bastion host

Page https://validatedpatterns.io/patterns/industrial-edge/getting-started/ says in step 7 of https://validatedpatterns.io[How to deploy](/patterns/industrial-edge/getting-started/#how-to-deploy) section

> You can deploy the pattern using the [Validated Patterns Operator](https://validatedpatterns.io/infrastructure/using-validated-pattern-operator/) directly. If you deploy the pattern using the Validated Patterns Operator, installed through `Operator Hub`, you will need to run `make load-secrets` through a terminal session on your laptop or bastion host.

However, running that yields
```
$ make load-secrets
make -f common/Makefile load-secrets
make[1]: Entering directory '/home/test/project/validatedpatterns/industrial-edge'
common/scripts/vault-utils.sh push_secrets industrial-edge
common/scripts/vault-utils.sh: line 31: ansible-playbook: command not found
make[1]: *** [common/Makefile:63: load-secrets] Error 127
make[1]: Leaving directory '/home/test/project/validatedpatterns/industrial-edge'
make: *** [Makefile:23: load-secrets] Error 2
```

Installation of Ansible is not listed in https://validatedpatterns.io/learn/quickstart/ linked from the second [Prerequisites](https://validatedpatterns.io/patterns/industrial-edge/getting-started/#prerequisites-1) section.

Does that dependency need to be documented? Or should that command instead be `./pattern.sh make load-secrets`?

--- a/content/patterns/industrial-edge/getting-started.md
+++ b/content/patterns/industrial-edge/getting-started.md
@@ -137,9 +137,9 @@ For installation tooling dependencies, see [Patterns quick start]({{< ref "/cont
    git push origin deploy-v2.3
    ```
 
-1. You can deploy the pattern using the [Validated Patterns Operator](/infrastructure/using-validated-pattern-operator/) directly. If you deploy the pattern using the Validated Patterns Operator, installed through `Operator Hub`, you will need to run `make load-secrets` through a terminal session on your laptop or bastion host.
+1. You can deploy the pattern using the [Validated Patterns Operator](/infrastructure/using-validated-pattern-operator/) directly. If you deploy the pattern using the Validated Patterns Operator, installed through `Operator Hub`, you will need to run `./pattern.sh make load-secrets` through a terminal session on your laptop or bastion host.
 
-1. If you deploy the pattern through a terminal session on your laptop or bastion host login to your cluster by using the`oc login` command or by exporting the `KUBECONFIG` file.
+1. If you deploy the pattern through a terminal session on your laptop or bastion host login to your cluster by using the `oc login` command or by exporting the `KUBECONFIG` file.
 
    ```sh
    oc login

